_maps/fortunapolis/Fortunapolis.yaml

```diff
@@ -0,0 +1,182 @@
+---
+name:
+  en: Fortunapolis
+desc:
+  en: A city where many come seeking the road to fortune, but who will be the first to find it? Vacant plots already included. 
+ruleSet: Standard
+theme: DragonQuest
+initialCash: 2200
+targetAmount: 22000
+baseSalary: 200
+salaryIncrement: 250
+maxDiceRoll: 7
+frbFile1: fortunapolis
+frbFile2: fortunapolis2
+background: bg901fixed
+mapIcon: fortunapolis
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/Fortunapolis.music.zip
+  - https://drive.google.com/file/d/1bkGFZpadHsJ56twXWFQVa50TslV6jNOk/view?usp=sharing

```

Comment:
This Google Drive download link should be: `https://drive.google.com/uc?export=download&confirm=1&id=1bkGFZpadHsJ56twXWFQVa50TslV6jNOk`